compiler/parser/chapel.ypp

```diff
@@ -521,15 +523,31 @@ stmt:
 | error TSEMI                          { $$ = buildErrorStandin(); }
 ;
 
+module_decl_start:
+  access_control opt_prototype TMODULE ident_def
+    {
+      $$ = buildModule($4, currentModuleType, NULL, yyfilename, $1, $2, @1.comment);
+      @$.comment = currentModuleName; // "push" prev module into comment field
```

Comment:
Discussed this with Lydia and added a new field with a more specialized name/purpose to address this concern.  A stack is the right way to go to get the full module path resolution discussed in issue #11825, but wasn't something I'd realized when focusing on the bugs filed in the issue that this PR fixes (and I think this will add lots of value until such time that #11825 is resolved... i.e., I don't think we should put off this simple fix while waiting for the more general solution).